examples/docs-wisdom-1/index.mjs

```diff
@@ -0,0 +1,28 @@
+import { loadStdlib } from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+
+const role = 'seller';
+console.log(`Your role is ${role}`);
+
+const stdlib = loadStdlib(process.env);
+console.log(`The consensus network is ${stdlib.connector}.`);
+
+const commonInteract = {};
+
+// Seller
+if (role === 'seller') {
+  const sellerInteract = { 
+    ...commonInteract };
+		
+  const acc = await stdlib.newTestAccount(stdlib.parseCurrency(1000));
+  const ctc = acc.contract(backend);
+  await backend.Seller(ctc, sellerInteract);
```

Comment:
You should use the new syntax... `await ctc.participants.Seller(sellerInteract)`